blog/llm-finetuning/README.md

```diff
@@ -0,0 +1,34 @@
+# LLM Finetuning using HuggingFace + Determined
+
+First install Determined on your local machine:
+```bash
+pip install determined
+```
+
+Then finetune:
+```bash
+det e create distributed.yaml . 
+```
+
+Change configuration options in `distributed.yaml`.
+
+Test your model's generation capabilities:
+
+```bash
+python test_model.py --exp_id <exp_id> --dataset_subset <dataset_subset>
```

Comment:
let's make sure to mention that exp_id comes from det e create distributed.yaml.